libs/core/dtos/time-interval.dto.ts

```diff
@@ -0,0 +1,15 @@
+/** Time interval DTO. */
+export type TimeIntervalDto = {
```

Comment:
```suggestion
export type DateTimeIntervalDto = {
```
Still, you work not only with time but also with the date